department-of-veterans-affairs/va.gov-cms
# VAGOV-4967: Improve TOC help text.

QA:
* Go to /node/add/health_care_region_detail_page 
* Check the TOC boolean field, the help text should be good.
